sysutils/squashfs-tools/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        plougher squashfs-tools 4.5
+checksums           rmd160  6543c75ac3313dd72aef9d951cdfb0211d299a1c \
+                    sha256  e3bc4d23fc8e5f1ede58ea6a438fc3454ffb8dc715a60b6e78b6a10e4dfbc19a \
+                    size    250518
+
+categories  sysutils
+platforms   darwin
+license     GPL-2+
+maintainers {mit.edu:quentin @quentinmit} \
+            openmaintainer
+description Tools to create and extract Squashfs filesystems
```

Comment:
Your indentation in the Portfile is rather inconsistent. This is not a very important issue but it would be nice if you could fix it. (Indentation in the patch also looks a bit inconsistent.) BTW, have you submitted the patch upstream?

Reply:
I haven't submitted the patch upstream. They don't have any form of configure script, so I suspect they aren't very interested in supporting non-Linux platforms (and they've closed some other portability PRs), but I can try. You can see the individual diffs at https://github.com/quentinmit/squashfs-tools/tree/darwin

I didn't notice the whitespace in the patch is inconsistent because my editor didn't pick up on the fact that they're using tabs for indentation, so it mixed 2-space tabs and spaces that look identical to me. I believe I've made it all consistent and matching local style now.